styles/ableplayer.css

```diff
@@ -394,6 +394,7 @@ input[type=range]:focus::-ms-fill-upper {
 	font-size: 0.9em;
 	background-color: transparent;
 	padding: 0.5em 0.5em 0.25em;
+	box-sizing: content-box; /* overrides the border-box set by openlearning.scss */
```

Comment:
We can probably leave the comment out, since this module might be used somewhere other than openlearning site by other people :)

Reply:
that's a good call 👍